tests/integration_tests/test_public_api.py

```diff
@@ -155,7 +155,7 @@ def test_public_api(cloud: BaseCloud):
     ],
     indirect=True,
 )
-def test_public_api_mininal_images(cloud: BaseCloud):
+def test_public_api_minimal_images(cloud: BaseCloud):
```

Comment:
I much prefer `mininal`.... But, if we prefer to be correct and all +1.

Reply:
Bahahahah